src/Resources/config/doctrine/File.orm.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
I needed to move this file to /uploader-bundle/Resources/config/doctrine/File.orm.xml to fix this error: 

```
In AbstractDoctrineExtension.php line 250:
  Specified non-existing directory "C:\SymfonyProjects\moyaplastApp\vendor\vich\uploader-bundle/Resources/config/doctrine" as Doctrine mapping source.
```

Maybe there are a problem?¿

Reply:
+1

Got the same issue here. It prevents the completion of my deployment. If I manually copy/move the file, it's ok, but on my production stack, I can't (and don't want to) manually change the files, everything being driven but git scripts.

There seems to be a mismatch between the two paths:

- `/uploader-bundle/Resources/config/doctrine/File.orm.xml` that the `AbstractDoctrineExtension.php` tries to reach 
- `/uploader-bundle/src/Resources/config/doctrine/File.orm.xml` where the file actually is

(note the `/src/` on the second path)

Anyone knows where to fix that so that the update to sf 5.4 doesn't break the bundle?